mcux/mcux-sdk/drivers/flexio/fsl_flexio.c

```diff
@@ -114,13 +114,19 @@ void FLEXIO_Init(FLEXIO_Type *base, const flexio_config_t *userConfig)
     FLEXIO_Reset(base);
 
     ctrlReg = base->CTRL;
+#if !(defined(FSL_FEATURE_FLEXIO_HAS_DOZE_MODE_SUPPORT) && (FSL_FEATURE_FLEXIO_HAS_DOZE_MODE_SUPPORT == 0))
```

Comment:
Have these changes been submitted back the the SDK team for inclusion in their backend?

Reply:
Yes David and they have been merged today.